pkg/operator/controller/ingress/namespace_test.go

```diff
@@ -0,0 +1,50 @@
+package ingress
+
+import (
+	"testing"
+
+	"github.com/openshift/cluster-ingress-operator/pkg/manifests"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+func TestRouterNamespaceChanged(t *testing.T) {
+	testCases := []struct {
+		description string
+		mutate      func(*corev1.Namespace)
+		expect      bool
+	}{
+		{
+			description: "if nothing changes",
+			mutate:      func(_ *corev1.Namespace) {},
+			expect:      false,
+		},
+		{
+			description: "if namespace workload annotation changes",
+			mutate: func(ns *corev1.Namespace) {
+				ns.Annotations["workload.openshift.io/allowed"] = "foo"
+			},
+			expect: true,
+		},
+		{
+			description: "if namespace network-policy annotation changes",
+			mutate: func(ns *corev1.Namespace) {
+				ns.Labels["policy-group.network.openshift.io/ingress"] = "foo"
+			},
+			expect: true,
+		},
```

Comment:
Please add a test case where an unmanaged annotation and an unmanaged label are changed.

Reply:
done.